src/paulscode/android/mupen64plusae/PlayMenuActivity.java

```diff
@@ -213,6 +210,22 @@ protected void onResume()
         mPrefs.registerOnSharedPreferenceChangeListener( this );
         mMogaController.onResume();
         refreshViews();
+        
+        if( ACTION_RESUME.equals( action ) )
```

Comment:
Yeah, that was the ultimate plan. Although I suppose it makes sense to move it now.